### PR TITLE
include thoth yaml file for kebechet updates

### PR DESCRIPTION
**Pull request by harshad16**
## Related Issues and Dependencies

- Setup Thoth-bots on the application

## This introduces a breaking change

- [x] No

## This Pull Request implements

- PR contains configuration file used by [kebechet](https://github.com/thoth-station/kebechet) bot.
- init script for kebechet to track the release of the application.

## Description

include `.thoth.yaml` file for kebechet updates
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      version: "8"
+    python_version: "3.6"
+    recommendation_type: latest
+
+managers:
+  - name: info
+  - name: version
+    configuration:
+      maintainers:
+        - vpavlin
```
Comment (harshad16) [inline .thoth.yaml:18]: maintainer list allows only allowed users to make a release request to kebechet.
Please add other here who might need permission to release the project with thoth-bots.

File: .thoth.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+host: khemenu.thoth-station.ninja
+tls_verify: false
+requirements_format: pipenv
```
Comment (vpavlin) [inline .thoth.yaml:3]: We don't use pipenv at the moment

File: .thoth.yaml (hunk 1 above)
Comment (vpavlin) [reply .thoth.yaml:18]: PLease add at least @LaVLaS we can figure out more later:)

File: .thoth.yaml (hunk 2 above)
Comment (harshad16) [reply .thoth.yaml]: Removed the Thoth-Advise input.

**Pull request merged**